synapse/metrics/__init__.py

```diff
@@ -502,6 +502,16 @@ def collect(self):
 
 last_ticked = time.time()
 
+# 3PID send info
+threepid_send_requests = Histogram(
+    "synapse_threepid_send_requests_with_tries",
+    documentation="Number of requests for a 3pid token by retry count. Note if"
```

Comment:
```suggestion
    documentation="Number of requests for a 3pid token by try count. Note if"
```

As this includes the initial try as well?